components/provisioner/pkg/gqlschema/schema.graphql

```diff
@@ -265,6 +265,7 @@ input UpgradeRuntimeInput {
 
 input UpgradeShootInput {
     gardenerConfig: GardenerUpgradeInput! # Gardener-specific configuration for the cluster to be upgraded
+    administrators: [String]                # List of administrators
```

Comment:
I'd suggest changing it to `administrators: [String!]`. We don't need to have `null` administrators